public/assets/js/scripts.js

```diff
@@ -1,17 +1,39 @@
-$(document).ready(function(){
+$(document).ready(function(){    
     // slick slider
     $('.benefits-slider').slick({
         speed: 500,
         infinite: true,
-        slidesToShow: 3,
-        slidesToScroll: 1,
+        centerMode: true,
         autoplay: true,
         autoplaySpeed: 2000,
-        centerPadding: '3rem',
+        slidesToScroll: 1,
+        centerPadding: '0',
+        slidesToShow: 3,
+        responsive: [
+            {
+              breakpoint: 991.98,
```

Comment:
Any special reason for using 991.98 here. Can we stick with bootstrap default devices and only use custom sizes when it is absolutely need. 
```
Small devices (landscape phones, 576px and up)
@media (min-width: 576px) { ... }

// Medium devices (tablets, 768px and up)
@media (min-width: 768px) { ... }

// Large devices (desktops, 992px and up)
@media (min-width: 992px) { ... }

// Extra large devices (large desktops, 1200px and up)
@media (min-width: 1200px) { ... }
```
https://getbootstrap.com/docs/4.0/layout/overview/